dash_app.py

```diff
@@ -3,7 +3,6 @@
 import pandas as pd
 
 import dash_bootstrap_components as dbc
-import dash_dataframe_table
```

Comment:
yes thank you for removing this! I thought I'd use it but I have to build the table manually instead of from a dataframe directly.